queries/agenda.js

```diff
@@ -30,6 +30,44 @@ const fetchFilesFromAgenda = async (agendaId) => {
   return parseSparqlResults(data);
 };
 
+const fetchFilesFromAgendaByMandatees = async (agendaId, mandateeIds) => {
+  const queryString = `
+  PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+  PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+  PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+  PREFIX dct: <http://purl.org/dc/terms/>
+  PREFIX dbpedia: <http://dbpedia.org/ontology/>
+  PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+  PREFIX nfo: <http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#>
+  PREFIX prov: <http://www.w3.org/ns/prov#>
+  PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+  SELECT DISTINCT (?file AS ?uri) ?name ?extension ?document ?documentName
+  WHERE {
+      ?agenda a besluitvorming:Agenda ;
+          mu:uuid ${sparqlEscapeString(agendaId)} ;
+          dct:hasPart ?agendaitem .
+      ?agendaitem a besluit:Agendapunt ;
+          besluitvorming:geagendeerdStuk ?document .
+      {
+        ?agendaitem ext:heeftBevoegdeVoorAgendapunt ?mandatee .
+        ?mandatee mu:uuid ?mandateeId .
+        FILTER (?mandateeId IN (${mandateeIds.map((id) => sparqlEscapeString(id)).join(', ')}))
+      } UNION {
+        FILTER NOT EXISTS { ?agendaitem ext:heeftBevoegdeVoorAgendapunt ?mandatee }
+      }
```

Comment:
Technically, this part of the query is extractable if we wanted.
It makes the query reusable with or without mandatee filter instead of 90% code duplication